cmd/kubelet/app/server.go

```diff
@@ -560,8 +560,30 @@ func SimpleKubelet(client *client.Client,
 // Eventually, #2 will be replaced with instances of #3
 func RunKubelet(kcfg *KubeletConfig, builder KubeletBuilder) error {
 	kcfg.Hostname = nodeutil.GetHostname(kcfg.HostnameOverride)
```

Comment:
I think you are breaking ppl using hostname_override now. Should we treat the value of flag hostname_override as node_name here?